qc/find_holes.py

```diff
@@ -199,7 +199,7 @@ def main(args):
 
     # should be as a terrain grid - but problems with high veg on fields!!!
     pc = pointcloud.fromAny(pargs.las_file).cut_to_class(cut_to)
-    pc_ref = pointcloud.fromAny(pargs.ref_data).cut_to_class(5)
+    pc_ref = pointcloud.fromAny(pargs.ref_data).cut_to_class(constants.terrain)

```

Comment:
use `pargs.class` here instead. Otherwise overriding the default value will have not effect.